### DISCUSSION TITLE
После сохранения кода в "Apps Script" предыдущий код утерян безвозвратно?

### DISCUSSION THREAD

**Discussion by Bondian** [Chimildic/goofy#190] [Помощь с ошибкой] [answered]
При обновлении библиотеки, вставил свежий код не туда и сохранил. Таким образом перезаписал свой нужный...
Уже не восстановить?

**Comment by Chimildic** [comment, marked as answer]
В редакторе кода нажмите "использовать классическую версию редактора" (справа сверху). Перейдите в нужный файл. Затем меню _Файл_ > _Смотреть историю версий_. 

Старый редактор в любой момент могут отключить. В новом такой функции не нашел. Так что бэкапы что ли сделайте, на всякий случай.

**Reply by Bondian** [reply]
Спасибо!
